content/js1/sprints/2/prep/index.md

```diff
@@ -8,4 +8,406 @@ backlog= 'Module-JS1'
+To convert to a percentage, we can multiply the number by 100 and then add a `"%"` sign on the end.
+
+So we can write:
+
+```js {linenos=table,linenostart=1}
+const num = 0.5;
+const percentage = `${num * 100}%`;
+```
+
+Recalling template literals, the expression in the curly braces will be evaluated first and then inserted into the string, giving us the percentage string.
+
+{{<note type="exercise" title="Exercise 1.1">}}
+
+The following variables represent the dimensions of a rectangle:
+
+```js
+height; // could have a value like 10
+width; // could have a value like 30
+```
+
+Write expressions for the following in terms of the variables above:
+a) area of the rectangle
+b) perimeter of the rectangle
+
+{{</note>}}
+
+{{<note type="exercise" title="Exercise 1.2">}}
+
+The following variable represents the cost of something in pence:
+
+```js
+price; // could have a value like "130p"
+```
+
+Write an expression for the price in pounds. The price in pounds should start with a "£" symbol. The price should be to 2 decimal places.
+
+{{</note>}}
+
+### ♻️ Wrapping up functionality
+
+Now we want to wrap up the expression we’ve created and reuse it to convert any number into a percentage.
+
+We can write a function declaration to reuse some code. A function declaration looks like this:
+
+```js {linenos=table,linenostart=1}
+function convertToPercentage() {}
+```
+
+The `function` declaration consists of the following syntactic elements:
+
+- `function` keyword, denotes the beginning of a function declaration
+- `convertToPercentage` - this is the name of the function
+- `{}` - the body of the function is written inside the curly braces
+
+We can wrap the functionality we want to reuse inside the curly braces by writing:
+
+```js {linenos=table,linenostart=1}
+function convertToPercentage() {
+  const num = 0.5;
+  const percentage = `${num * 100}%`;
+}
+```
+
+### 🔎 Checking the output
+
+Earlier we saw how to store a function's output in a variable.
+
+```js
+const result = Math.round(10.3);
+```
+
+In the code above, we call `Math.round` which takes the input `10.3` and then outputs the rounded number `10`. So `result` stores a value of `10`.
+
+Now we want to call the function `convertToPercentage` and store the output in a variable.
+We can store the return value in a variable in exactly the same way:
+
+```js {linenos=table,hl_lines=["5"],linenostart=1}
+function convertToPercentage() {
+  const num = 0.5;
+  const percentage = `${num * 100}%`;
+}
+const result = convertToPercentage(0.3);
+```
+
+Then we can print the value of the result to the terminal using `console.log`.
+
+```js {linenos=table,hl_lines=["6"],linenostart=1}
+function convertToPercentage() {
+  const num = 0.5;
+  const percentage = `${num * 100}%`;
+}
+const result = convertToPercentage(0.3);
+console.log(result);
+```
+
+If our function behaves as intended, we expect the output to be `"30%"`.
```

Comment:
Do we want to say this given the exercise below? Maybe _after_ the exercise?

Also, can we differentiate between what `result` evaluates to (`"30%"`) and what's printed to the console (`30%` without quotes)